blueman/bluez/obex/ObjectPush.py

```diff
@@ -14,20 +17,20 @@ class ObjectPush(Base):
 
     _interface_name = 'org.bluez.obex.ObjectPush1'
 
-    def __init__(self, session_path):
+    def __init__(self, session_path: str):
         super().__init__(interface_name=self._interface_name, obj_path=session_path)
 
-    def send_file(self, file_path):
-        def on_transfer_started(transfer_path, props):
+    def send_file(self, file_path: str) -> None:
+        def on_transfer_started(transfer_path: str, props: Dict[str, str]) -> None:
             logging.info(" ".join((self.get_object_path(), file_path, transfer_path)))
             self.emit('transfer-started', transfer_path, props['Filename'])
 
-        def on_transfer_error(error):
+        def on_transfer_error(error: BluezDBusException) -> None:
             logging.error("%s %s" % (file_path, error))
             self.emit('transfer-failed', error)
 
         param = GLib.Variant('(s)', (file_path,))
         self._call('SendFile', param, reply_handler=on_transfer_started, error_handler=on_transfer_error)
 
-    def get_session_path(self):
-        return self.get_object_path()
+    def get_session_path(self) -> str:
+        return cast(str, self.get_object_path())
```

Comment:
I updated this as `cast` would prevent actual type checking.

mypy complains if we return type `Any` because `str` is expected.

As long as the type is `Any` / unknown both `cast(str, ...)` and `v: str = ...` make it `str` but considering that we put some stubs into place that define a useful type, mypy would check it for `v: str = ...` (e.g. report an error if the type is `int`) but not for `cast(str, ...)`, so yes, we should avoid `cast` wherever possible.